### ISSUE TITLE
Rotation doesn't work properly

### ISSUE THREAD

**Issue by 92flash** [openboard-team/openboard#587]
Hi. Lately I'm having problems with Openboard not rotating properly. It then stays in landscape or in portrait mode. After force stopping Openboard it works properly again, but after a few hours it goes back to having the same problems. Is this something that can be fixed? I'm on the latest version of GrapheneOS stable.

![ce89b5e4-d86b-44d9-b8a3-160e10a2c41d](https://user-images.githubusercontent.com/76122930/161763190-cf2eaea2-89a5-4698-8a6b-040b8b56b616.png)

![0acd6b3d-1946-4524-9e39-4c407474f98f](https://user-images.githubusercontent.com/76122930/161763193-cbcd99f9-66b3-4071-90a3-941f392715db.png)

**Comment by stakillion** [comment]
This is exactly the problem I've been dealing with: The keyboard gets stuck in either landscape or portrait mode and becomes unusable when the screen is rotated the other way. It's a real bummer because this is otherwise the perfect keyboard for me. I'm on GrapheneOS as well.

**Comment by 92flash** [comment]
@nstana Yes, I agree. I've tried multiple keyboard applications from F-droid, but none have the perfect balance of features Openboard has. If I have this problem I either decide to deal with it until it resolves itself, sometimes a few hours or day later, or force stop the app, and set Openboard as the keyboard again.
